addon/helpers/list.js

```diff
@@ -2,6 +2,18 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array instanceof Array) {
+      return array;
+    } else if (array && typeof array !== 'string' && array.length != null) {
+      return array;
```

Comment:
Let's copy the "enumerable" to an array here so that the microstate is always working with one. Otherwise, if we do pass it a file list it will break since it doesn't have methods like `forEach` and `concat`